src/Throttle/Throttler/ElasticWindowThrottler.php

```diff
@@ -73,8 +74,9 @@ public function access()
     public function hit()
     {
         $this->counter = $this->count() + 1;
+        $item = new CacheCount($this->counter, $this->ttl);
 
-        $this->cache->set($this->key, $this->counter, $this->ttl);
+        $this->cache->setItem($this->key, $item);
```

Comment:
The `item` variable is used only once, so this can be done inline. Applies to a number of similar cases below

Reply:
👍